pallets/registry/src/weights.rs

```diff
@@ -0,0 +1,93 @@
+// This file is part of HydraDX-node.
```

Comment:
what is the purpose of this file? is it still WIP?

Reply:
benchmarks in progress. It will be completed today and all pallets ( not only registry) will be rebenchmarked

Reply:
So far PR looks good, just a few cosmetic changes. Will get back to it next week then.